src/schema/v2/me/index.ts

```diff
@@ -151,6 +155,7 @@ const MeField: GraphQLFieldConfig<void, ResolverContext> = {
       "followsAndSaves",
       "lotsByFollowedArtistsConnection",
       "identityVerification",
+      "identityVerified",
```

Comment:
I had to remove this field to make it work otherwise MP wouldn't attempt to load data using `meLoader`.